src/cmd-import

```diff
@@ -231,10 +231,12 @@ def skopeo_inspect(image):
 def parse_timestamp(timestamp):
     # datetime's doesn't support nanoseconds.
     # So trim it.
-    if len(timestamp) > 26 and timestamp[19] == '.':
-        timestamp = timestamp[:26] + "Z"
+    # ignore the microseconds too, as not all images has it.
+    timestamp, _, _ = timestamp.partition('.')
+    if not timestamp.endswith('Z'):
+        timestamp += 'Z'
 
-    timestamp = datetime.datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%S.%fZ')
+    timestamp = datetime.datetime.strptime(timestamp, '%Y-%m-%dT%H:%M:%SZ')
     return rfc3339_time(timestamp.replace(tzinfo=datetime.timezone.utc))
```

Comment:
Hmm, `timestamp.replace(tzinfo=...)`... isn't that _replacing_ the timezone instead of _moving_ the timestamp to UTC?

I think in practice it likely doesn't matter because most things will have a UTC stamp, but note the OCI spec just mandates an RFC3339 timestamp only, and those can have differing timezones.